### PR TITLE
changelog updated for Releasing Core 1.0.0-beta.4 and Identity 1.0.0-beta.2

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2021-1-11)
```
Comment (RickWinter) [inline sdk/core/azure-core/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.4 (2021-1-12)
```

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-1-11)
```
Comment (RickWinter) [inline sdk/identity/azure-identity/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.2 (2021-1-12)
```

**Review by RickWinter — APPROVED** [review]

**Comment by ahsonkhan** [comment]
> changelog updated for Releasing Core 1.0.0-beta.3

Shouldn't the PR title be `1.0.0-beta.4`?

**Review by ahsonkhan — COMMENTED** [review]
We should review the contents of the changelog as well.

**Review by ahsonkhan — APPROVED** [review]
Other than updating the dates, looks good.

**Review by ahsonkhan — APPROVED** [review]

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-1-12)
```
Comment (ahsonkhan) [inline sdk/identity/azure-identity/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.2 (2021-1-13)
```

File: sdk/core/azure-core/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2021-1-12)
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.4 (2021-1-13)
```

**Pull request merged**